Dfe.Academies.External.Web/Models/ConversionApplication.cs

```diff
@@ -4,6 +4,11 @@ namespace Dfe.Academies.External.Web.Models;
 
 public class ConversionApplication
 {
+    public ConversionApplication()
+    {
+        SchoolOrSchoolsApplyingToConvert = new();
```

Comment:
This can be initialised on line 20, I think that's a bit neater?

`    public List<SchoolOrSchoolsApplyingToConvert> SchoolOrSchoolsApplyingToConvert { get; set; } = new();
`

Reply:
yes, great idea